cloud_computing/migrating_app_to_cloud/app_remote_storage.md

```diff
@@ -0,0 +1,248 @@
+
+- **Default region name:** This is the AWS region where you want your AWS services to operate. Examples include:
+
+  - `eu-west-1` (Ireland)
+  - `eu-west-2` (London)
+  - `eu-west-3` (Paris)
+
+- **Default output format:** Choose how you want the CLI to display the results. Options include json, text, and table.
+
+   Example input:
+
+   ```bash
+   AWS Access Key ID [None]: AAKIEXAMPLE
+   AWS Secret Access Key [None]: wJalFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+   Default region name [None]: eu-west-2
+   Default output format [None]: json
+   ```
+
+#### Step 3: Test the Configuration
+
+To verify that the AWS CLI is correctly configured, try listing your S3 buckets:
+
+```bash
+aws s3 ls
+```
+
+If the configuration is successful, you will see a list of your S3 buckets. If there are any issues, you will likely see an error message indicating the problem.
+
+
+### Setting Up AWS Resources
+
+Before cloning and running the application, you need to create an S3 bucket and a DynamoDB table on AWS, as the application will use these resources for storage and metadata management.
+
+#### 1. **Creating an S3 Bucket:**
+
+The S3 bucket will be used to store the images uploaded by the users and processed by the application.
+
+1. Open the AWS Management Console and navigate to the [S3 service](https://s3.console.aws.amazon.com/s3/home). Please make sure you are in the correct AWS region. You can select the region by clicking on the region dropdown in the top right corner of the console. Refer to the image below, where the region selection is highlighted with a red box for guidance on changing the region
+
+   ![AWS Region Selection](S3_Region.jpg)
+
+2. Click on **Create bucket**. Refer to the image below if you need guidance, where the "Create bucket" option is highlighted with a red box:
+
+   ![Where is S3 Create Button](Where_S3_Create_Button.jpg)
+
+3. Provide a unique bucket name (e.g., `ar-image-processing-bucket`). It's recommended to include your initials or another unique identifier in the bucket name to avoid naming conflicts. Refer to the image below, where the field for entering the bucket name is highlighted with a red box:
+
+   ![S3 Bucket Name Input](Create_S3_Bucket.jpg)
+
+4. Leave the other settings as default unless you have specific requirements. However, it's important to understand and consider the following options:
+
+#### Object Ownership:
+
+- **Object Ownership** defines the access control behavior for objects uploaded to the bucket. By default, Amazon disables Access Control Lists (ACLs), which is the recommended approach for most use cases as it simplifies permissions management. Disabling ACLs ensures that the bucket owner automatically owns all objects, even if they are uploaded by different AWS accounts.
+
+#### Block Public Access:
```

Comment:
This information is repeated in `app_migrate_to_aws.md`. I would keep info in one of them and link to that.